src/test/java/org/jasig/portlet/newsreader/processor/RomeNewsProcessorImplTest.java

```diff
@@ -61,7 +58,7 @@ public void testParsing() throws IOException, IllegalArgumentException, FeedExce
         InputStream in = testFeed.getInputStream();
         NewsFeed feed = processor.getFeed(in, "antisamy-textonly", "antisamy-textonly");
         
-        assertEquals(20, feed.getEntries().size());
+        assertEquals(10, feed.getEntries().size());
```

Comment:
If we've removed the maxStories preference, is this method even testing anything?